### DISCUSSION TITLE
Dark mode?

### DISCUSSION THREAD

**Discussion by Andy76swe** [LibreHardwareMonitor/LibreHardwareMonitor#1502] [Ideas]
Would be nice if you could add a dark mode to the app ;)

**Comment by Taiizor** [comment]
Ogh (;

![image](https://github.com/user-attachments/assets/a0affb6b-4769-4cbc-aa72-4d9d51ee915b)

**Reply by Andy76swe** [reply]
I have latest version 0.9.3 and there is no such menu

**Reply by Taiizor** [reply]
The last release build was taken in 2023. I’ve taken a build for you. You can download and use it from the link below:

[LibreHardwareMonitor-net472.zip](https://github.com/user-attachments/files/17533370/LibreHardwareMonitor-net472.zip)
